controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -58,7 +59,8 @@ const nodePolicySyncEventName = "node-policy-sync-event"
 // SriovNetworkNodePolicyReconciler reconciles a SriovNetworkNodePolicy object
 type SriovNetworkNodePolicyReconciler struct {
 	client.Client
-	Scheme *runtime.Scheme
+	Scheme      *runtime.Scheme
+	FeatureGate featuregate.FeatureGate
```

Comment:
This field is not used. I guess it will be used in some future PR, right?

Reply:
Yes, this is preparation for the next PR.